snap/snapcraft.yaml

```diff
@@ -140,36 +146,11 @@ parts:
       usr/share/doc/prometheus-mysqld-exporter/copyright: licenses/COPYRIGHT-prometheus-mysqld-exporter
       usr/share/doc/prometheus-mysqlrouter-exporter/copyright: licenses/COPYRIGHT-prometheus-mysqlrouter-exporter
       usr/share/doc/percona-xtrabackup/LICENSE.gz: licenses/LICENSE-percona-xtrabackup
+      usr/share/doc/mysql-audit-log-filter-plugin/copyright: licenses/COPYRIGHT-mysql-audit-log-filter-plugin
+      usr/share/doc/mysql-audit-log-plugin/copyright: licenses/COPYRIGHT-mysql-audit-log-plugin
+      usr/share/doc/mysql-auth-ldap-plugin/copyright: licenses/COPYRIGHT-mysql-auth-ldap-plugin
+      usr/share/doc/mysql-binlog-utils-udf-plugin/copyright: licenses/COPYRIGHT-mysql-binlog-utils-udf-plugin
```

Comment:
Q: do we need to copy LICENSE ?
Is it the same as `usr/share/doc/percona-xtrabackup/LICENSE.gz` ? Anyway IMHO, copy it again as the one above is `LICENSE-percona-xtrabackup`.

Reply:
there was no license for the percona server plugins (just copyright files) - in this case, we decided to copy the COPYRIGHT file instead

to clarify: is the recommendation to copy the percona xtrabackup license for the the plugins? i believe it may be prudent to have a copy of the COPYRIGHT file since this file can be changed at any point by percona